modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -48,9 +50,11 @@ The API carousel component shows a horizontally scrollable set of API cards.
 * *Card Border*: Set the thickness of the card borders.
 * *Card Border Radius*: Set the radius of the card borders. A larger radius makes the cards look more rounded.
 * *Card Shadow*: Set the shadow beneath each card.
-* *Message If Empty*: Show this message if the carousel has no APIs. Default is "There are no APIs available right now."
+* *Message If Empty*: Show this message if the carousel has no APIs. Default is `There are no APIs available right now.`
 * *Filter Condition (Advanced)*: Filter APIs using package names, field names such as "API name" or "API description", and comparison of strings to those fields.
```

Comment:
"API description,"

GLOBALLY put commas and periods within quotation marks unless they might be confused as part of the quoted material (rare).

Reply:
Fixed.